dynet/cfsm-builder.h

```diff
@@ -38,7 +38,12 @@ class StandardSoftmaxBuilder : public SoftmaxBuilder {
   expr::Expression neg_log_softmax(const expr::Expression& rep, unsigned wordidx);
   unsigned sample(const expr::Expression& rep);
   expr::Expression full_log_distribution(const expr::Expression& rep);
-
+  std::vector<ParameterStorage*> get_parameters() {
```

Comment:
In this and the other classes in the commit, wouldn't be easier to save `local_model` as a member of the class, then when `get_parameters()` is called, call `local_model.get_parameters()`? The ability to do this simply is one of the major advantages of the `ParameterCollection` concept I think.